setup.py

```diff
@@ -538,7 +538,8 @@ def pxd(name):
                      '_libs/tslibs/ccalendar',
                      '_libs/tslibs/timedeltas',
                      '_libs/tslibs/timezones',
-                     '_libs/tslibs/nattype'],
+                     '_libs/tslibs/nattype',
+                     '_libs/tslibs/offsets'],
```

Comment:
@jreback I thought we had made the "pxdfiles" obsolete when moving to `cythonize`.  Am I remembering wrong?

Reply:
Never mind, thought I was commenting on an open PR, not an old one.